pkgs/applications/version-management/fossil/default.nix

```diff
@@ -17,11 +17,11 @@
 
 stdenv.mkDerivation rec {
```

Comment:
Since we are manually editing this derivation, let's use this opportunity to use the `finalAttrs` pattern.

Let me share a couple of useful links regarding this:

1. History: https://discourse.nixos.org/t/avoid-rec-expresions-in-nixpkgs/8293
2. Documentation: https://nixos.org/manual/nixpkgs/unstable/#mkderivation-recursive-attributes
3. Recent example of implementation: https://github.com/NixOS/nixpkgs/compare/17f96f7b978e61576cfe16136eb418f74fab9952..9e6ea843e473d34d4f379b8b0d8ef0425a06defe

Cheers.

Reply:
Gracias!